doc/source/tune/_tutorials/tune-sklearn.py

```diff
@@ -0,0 +1,147 @@
+
+To start out, change the import statement to get tune-scikit-learn’s grid search cross validation interface:
+
+"""
+# from sklearn.model_selection import GridSearchCV
+from tune_sklearn import TuneGridSearchCV
+
+#######################################################################
+# And from there, we would proceed just like how we would in Scikit-Learn’s interface!
+#
+# The `SGDClassifier`_ has a ``partial_fit`` API, which enables it to stop fitting to the data for a certain hyperparameter configuration. If the estimator does not support early stopping, we would fall back to a parallel grid search.
+
+# Other imports
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import SGDClassifier
+from sklearn import datasets
+import numpy as np
+
+# Load in data
+digits = datasets.load_digits()
+
+# Set training and test sets
+x = digits.data
+y = digits.target
+x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=.2)
+
+# Example parameters to tune from SGDClassifier
+parameter_grid = {"alpha": [1e-4, 1e-1, 1], "epsilon": [0.01, 0.1]}
+
+#######################################################################
+# As you can see, the setup here is exactly how you would do it for Scikit-Learn. Now, let's try fitting a model.
+
+tune_search = TuneGridSearchCV(
+    SGDClassifier(),
+    parameter_grid,
+    scheduler="MedianStoppingRule",
+    early_stopping=True,
+    max_iters=10)
+
+import time  # Just to compare fit times
+start = time.time()
+tune_search.fit(x_train, y_train)
+end = time.time()
+print("Tune Fit Time:", end - start)
+
+#######################################################################
+# Note the slight differences we introduced above:
+#
+#  * a `scheduler`, and
+#  * a specification of `max_iters` parameter
+#
+# The ``scheduler`` determines when to stop early - MedianStoppingRule is a great default, but see :ref:`Tune's documentation on schedulers <tune-schedulers>` here for a full list to choose from. ``max_iters`` is the maximum number of iterations a given hyperparameter set could run for; it may run for fewer iterations if it is early stopped.
+#
+# Try running this compared to the GridSearchCV equivalent, and see the speedup for yourself!
+
+from sklearn.model_selection import GridSearchCV
+# n_jobs=-1 enables use of all cores like Tune does
+sklearn_search = GridSearchCV(SGDClassifier(), parameter_grid, n_jobs=-1)
+
+start = time.time()
+sklearn_search.fit(x_train, y_train)
+end = time.time()
+print("Sklearn Fit Time:", end - start)
+
+###################################################################
+# Using Bayesian Optimization
+# ---------------------------
+#
+# In addition to the grid search interface, tune-sklearn also provides an interface, TuneSearchCV, for sampling from **distributions of hyperparameters**.
+#
+# In addition, you can easily enable Bayesian optimization over the distributions in only 2 lines of code:
+
```

Comment:
Might be worth noting that the user needs to `pip install bayesian-optimization`. This will pop up as a warning to the user and won't run the search if it isn't installed

Reply:
done